app/assets/javascripts/student_profile/profile_details.jsx

```diff
@@ -197,6 +238,39 @@ import _ from 'lodash';
       );
     },
 
+    renderSectionDetails: function() {
+      const sections = this.props.sections;
+      
+      // If there are no sections, don't generate the section
+      if (!sections || sections.length == 0) return null;
+
+      // If this student is not a high school student, don't generate the section
+      if (this.props.student.school_type != 'HS') return null;
+      
+      const columns = [
```

Comment:
This and the related methods in this component seem like they might split out nicely to a `SectionsTable` component or something like that, to keep it isolated and keep this component from getting huger :)